include/time.h

```diff
@@ -64,28 +64,38 @@
  * forward and backward as the system time-of-day clock is changed.
  */
 
-#define CLOCK_REALTIME     0
+#define CLOCK_REALTIME            0
 
 /* Clock that cannot be set and represents monotonic time since some
  * unspecified starting point. It is not affected by changes in the
  * system time-of-day clock.
  */
 
-#define CLOCK_MONOTONIC    1
+#define CLOCK_MONOTONIC           1
+
+/* Clock that measures CPU time consumed by this process (i.e., CPU
+ * time consumed by all threads in the process).
+ */
+
+#define CLOCK_PROCESS_CPUTIME_ID  2
+
+/* Clock that measures CPU time consumed by this thread */
+
+#define CLOCK_THREAD_CPUTIME_ID   3
 
 /* Monotonic system-wide clock that includes time spent in suspension. */
 
-#define CLOCK_BOOTTIME     2
+#define CLOCK_BOOTTIME            7
```

Comment:
Why to use 7 instead of 4? Are you planing to add later symbols from 4 to 6 ?

Reply:
Follow the Linux definition here:
https://github.com/torvalds/linux/blob/master/include/uapi/linux/time.h#L56

Reply:
I think we should not blindly follow Linux. Let's define to 4 and than redefine later if we need to add other clock types. I mean, what do we gain from getting the same numbers? Only if Linux apps are using pure values instead of defines and we what to run those without modifications.
I even would vote to leave values as they are and add `CLOCK_PROCESS_CPUTIME_ID` and `CLOCK_THREAD_CPUTIME_ID` as `3` and `4`